packages/core/integration-tests/test/javascript.js

```diff
@@ -569,7 +569,7 @@ describe('javascript', function() {
     ]);
   });
 
-  it('should support bundling workers of type module', async function() {
+  it.skip('should support bundling workers of type module', async function() {
```

Comment:
a TODO?

Reply:
Yeah will be addressed in the next PR for module/nomodule